Samples/All/BumbleRumble/Common/NetworkManager.cpp

```diff
@@ -1224,3 +1199,74 @@ PartyChatControl* NetworkManager::GetChatControl(std::string& peer)
     }
     return nullptr;
 }
+
+void NetworkManager::SetCognitiveServicesEnabled(bool bEnabled)
+{
+	m_enableCognitiveServices = bEnabled;
+
+	// For the purposes of the sample, enable or disable all possible features
+	SetTextChatTranslationOptions(bEnabled);
+	SetVoiceChatTranscriptionOptions(bEnabled, bEnabled, bEnabled, false, bEnabled); 
+}
+
+void NetworkManager::SetTextChatTranslationOptions(bool bTranslateToLocalLanguage)
+{
+    DEBUGLOG("SetTextChatTranslationOptions: bTranslateToLocalLanguage = %s", bTranslateToLocalLanguage ? "Enabled" : "Disabled");
+
+    // Enable translation to local language in chat controls.
+    PartyError err = m_localChatControl->SetTextChatOptions(
+        bTranslateToLocalLanguage ? PartyTextChatOptions::TranslateToLocalLanguage : PartyTextChatOptions::None,
+        nullptr
+    );
+
+    if (PARTY_FAILED(err))
+    {
+        DEBUGLOG("SetTextChatOptions failed: %s\n", GetErrorMessage(err));
+    }
+}
+
+void NetworkManager::SetVoiceChatTranscriptionOptions(bool bTranscribeSelf, bool bTranscribeOtherChatControlsWithMatchingLanguages, bool bTranscribeOtherChatControlsWithNonMatchingLanguages, bool bDisableHypothesisPhrases, bool bTranslateToLocalLanguage)

```

Comment:
see comment above.

Reply:
I wrote this function in this way so that anyone who reads the function knows that they can toggle each of the features and it isn't an all or nothing functionality